qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VerifyLoginSteps.java

```diff
@@ -40,6 +40,9 @@ public void verifyModulesAvailableOnHomePage() {
         assertTrue(homePage.isDataManagementAppPresent());
         assertTrue(homePage.isConfigureMetadataAppPresent());
         assertTrue(homePage.isSystemAdministrationAppPresent());
+        assertTrue(homePage.isDataManagementAppPresent());
+        
+
```

Comment:
any reason for adding the empty lines here?

Reply:
Resolved